controller-haproxy.go

```diff
@@ -108,9 +108,9 @@ func (c *HAProxyController) updateHAProxy() error {
 	}
 	needsReload = needsReload || reload
 
-	reload, err = c.handleDefaultService()
-	LogErr(err)
-	needsReload = needsReload || reload
+	// reload, err = c.handleDefaultService()
+	// LogErr(err)
+	// needsReload = needsReload || reload
```

Comment:
This commented block is to suppress error messages going into the logs.  The default assumption of the haproxy-controller (kubernetes-ingress) is that there is a POD that serves as a "default" web "we cannot find what you are looking for" when HTTP/HTTPS requests come into the proxy and don't match any server definitions.  Since we aren't using the HTTP/HTTPS features I didn't install any of the "default backend" support, thus the "handler" for that service fails and logs the same failure line over and over again.